src/js/components/Spinner/Spinner.js

```diff
@@ -68,7 +75,28 @@ const Spinner = forwardRef(
         }))
       : borderThemeProp;
 
-    // children will take precedence over theme attributes
+    // check the size of the pad plus border
+    let spinnerPadBorder;
+    if (!rest.border) {
+      if (size !== undefined) {
```

Comment:
same as above, we can simplify this logic if size defaults to `small` -- it seems we document it as that (https://v2.grommet.io/spinner#size) so i'd consider it a bug we're fixing

Reply:
agree I can do that just wanted to get thoughts on this direction